internal/pkg/skuba/deployments/ssh/ssh.go

```diff
@@ -188,7 +188,7 @@ func readerStreamer(reader io.Reader, outputChan chan<- string, description stri
 	for scanner.Scan() {
 		result.Write([]byte(scanner.Text()))
 		if description == "stdout" && !silent {
-			klog.V(1).Infof("%s", scanner.Text())
+			klog.V(2).Infof("%s", scanner.Text())
```

Comment:
Again, slight security paranoia flag here as we are logging all the output from the ssh command.

Reply:
I believe, most of logs comes from this line. Good to have differentiate log level.
I was confusing v(2) and V(3). I read doc again. klog.V(2) - Useful steady state information about the service and important log messages that may correlate to significant changes in the system. I think this satisfies V(2).